zubbi/models.py

```diff
@@ -39,6 +39,13 @@
     "verify_mode": "CERT_REQUIRED",
 }
 
+# We want to boost following fields
+SEARCH_BOOST_FIELDS = {
+    "role_name": "role_name^1.5",
+    "job_name": "job_name^1.5",
+    "description": "description^1.2",
```

Comment:
Did you test if this works as expected or could you point me to some documentation? I just couldn't find anything about this syntax in regard to boosting, apart from an old Stackoverflow answer which points to an outdated documentation: https://stackoverflow.com/a/24524040

Reply:
I have tested it works as expected, the syntax is mentioned also here: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html